redbadger/website-honestly
# Change events banner to just-in-time UXD webinar

### Motivation

We have a new webinar, which should be advertised with the banner on top of our events page.

### Test plan

- Go to events page.
- Make sure the new banner is visible in mobile, tablet and desktop view.
- Click on banner and make sure the correct webinar registration (just-in-time UXD) opens in a model dialog.

### Pre-merge checklist
